Client/Frontend/Browser/URLBarView.swift

```diff
@@ -524,9 +489,8 @@ class URLBarView: UIView {
         self.menuButton.isHidden = !self.toolbarIsShowing || inOverlayMode
         self.forwardButton.isHidden = !self.toolbarIsShowing || inOverlayMode
         self.backButton.isHidden = !self.toolbarIsShowing || inOverlayMode
-        self.shareButton.isHidden = !self.toolbarIsShowing || inOverlayMode
+        self.tabsButton.isHidden = !self.toolbarIsShowing || inOverlayMode
```

Comment:
So much `self` in the old code! I am now glad you didn't go and change existing usages and kept the diff to a minimum. I'll stick to self nits on lines that are added.

Reply:
Going to leave this in to keep things consistent. This class could use some cleanup once we are done.